plugins/gatsby-source-wagtail/schema/types.gql

```diff
@@ -41,5 +41,6 @@ module.exports = `
   type document {
     display_text: String
     document: String
+    url: String
```

Comment:
Does this support the fact that documents will have a document *or* a URL? Just making sure this doesn't enforce both.

Reply:
It does. if one field doesn't appear on the type, it will just be `null`. (for  knowledge's sake, `String!` would indicate a required field).